pages/chapter2.tsx

```diff
@@ -0,0 +1,41 @@
+import PageCard from '@/components/PageCard';
+import '../app/globals.css';
+import Layout from '@/components/Layout';
+import { chapter } from '@/constants/chapter';
+
+const chapterId = 2;
+
+const chapterMatch = chapter.find((chapter) => chapter.id === chapterId);
+
+if (!chapterMatch) {
+  throw new Error(`Chapter ${chapterId} not found`);
+}
+
+const { title, subtitle, details, devOscar, devJon } = chapterMatch;
+
+const chapter2 = () => {
```

Comment:
```js
export default function chapter2() {
```

Let's use this for all files

Reply:
Why do we change to a function declaration? Changes has been applied and waiting for your comments

Reply:
Replied in previous comment

Reply:
got it thank you